lib/api.js

```diff
@@ -54,9 +56,10 @@ Api.prototype.load = function () {
 }
 
 Api.prototype.isNodeIntegrationEnabled = function () {
-  return this.app.client.execute(function (requireName) {
+  var self = this
+  return self.app.client.execute(function (requireName) {
```

Comment:
same
```suggestion
  return this.app.client.execute(function (requireName) {
```

Reply:
Unfortunately I was getting test failure on windows without this :/